packages/components/src/Typeahead/Typeahead.component.js

```diff
@@ -35,11 +36,13 @@ function getItems(items, dataFeature) {
  *
  * <Typeahead {...props} />
  */
-function Typeahead({ onToggle, icon, position, docked, ...rest }) {
+function Typeahead({ onToggle, icon, position, docked, manageNavigation, ...rest }) {
```

Comment:
Could we not pass a flag but base this assertion with the presence of the onKeyDown prop ?

Reply:
I wanted to keep the opportunity to add additional behaviours to the handleKey but yes, you're right, it's more elegant like that. I push a fix, thanks